bokehjs/src/lib/models/widgets/file_input.ts

```diff
@@ -70,10 +98,11 @@ export abstract class FileInput extends Widget {
     this.prototype.default_view = FileInputView
 
     this.define<FileInput.Props>({
-      value:     [ p.String, '' ],
-      mime_type: [ p.String, '' ],
-      filename:  [ p.String, '' ],
+      value:     [ p.Any, [] ],
+      mime_type: [ p.Any, [] ],
+      filename:  [ p.Any, [] ],
```

Comment:
defaults for all these need to match the Python defatult, which is the empty string. Since the default mode is `multiple=False` that makes sense.

Reply:
I simply have no real grips on TS/JS and therefore looked up other examples.
I can do that, if the compiler accepts.

Reply:
Done in #9732